app-ios/Modules/Package.swift

```diff
@@ -209,6 +210,7 @@ package.targets = package.targets.map { target -> Target in
             target.plugins = []
         }
         target.plugins?.append(.plugin(name: "SwiftLintPlugin", package: "SwiftLint"))
+        target.plugins?.append(.plugin(name: "SwiftFormatPlugin", package: "swift-format"))
```

Comment:
`SwiftFormatPlugin` isn't a correct name 🙏🏼 
Please use `FormatPlugin` 🙏🏼 

https://github.com/apple/swift-format/blob/508.0.1/Package.swift#L37

And format or format-lint isn't succeeded. Can you check it? 🙇🏼